### DISCUSSION TITLE
How to divide Members

### DISCUSSION THREAD

**Discussion by DavidNaizheZhou** [Dlubal-Software/RFEM_Python_Client#116] [Q&A] [answered]
Hi, 

is there a function to divide members by distance or n?

Best regards, David

**Comment by OndraMichal** [comment, marked as answer]
Hi @DavidNaizheZhou,
unfortunately we don't have any of the right-click menu functions yet. But that's in the future. This is because it is very easy to execute programmatically in each script by simply adding one point to the desired location and changing the first and last points in the member. This applies to all manipulations.
